ci/kind/test-multi-cluster.sh

```diff
@@ -0,0 +1,164 @@
+#!/usr/bin/env bash
+
+# Copyright 2020 Antrea Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# The script runs kind e2e tests with different traffic encapsulation modes.
+
+set -eo pipefail
+
+function echoerr {
+    >&2 echo "$@"
+}
+
+_usage="Usage: $0 [--help|-h]
+        --help, -h                    Print this message and exit.
+"
+
+function print_usage {
+    echoerr -n "$_usage"
+}
+
+
+TESTBED_CMD=$(dirname $0)"/kind-setup.sh"
+FLOW_VISIBILITY_CH_ONLY_CMD=$(dirname $0)"/../../hack/generate-manifest.sh --no-grafana --node-port --secure-connection"
+CH_OPERATOR_YML=$(dirname $0)"/../../build/charts/theia/crds/clickhouse-operator-install-bundle.yaml"
+
+WORKDIR=$HOME
+MULTICLUSTER_KUBECONFIG_PATH=$WORKDIR/.kube
+EAST_CLUSTER_CONFIG="--kubeconfig=$MULTICLUSTER_KUBECONFIG_PATH/east"
+WEST_CLUSTER_CONFIG="--kubeconfig=$MULTICLUSTER_KUBECONFIG_PATH/west"
+CLUSTER_NAMES=("east" "west")
+
+function quit {
+  result=$?
+  if [[ $setup_only || $test_only ]]; then
+    exit $result
+  fi
+  echoerr "Cleaning testbed"
+  $TESTBED_CMD destroy east
+  $TESTBED_CMD destroy west
+}
+
+while [[ $# -gt 0 ]]
+do
+key="$1"
+
+case $key in
```

Comment:
A case for coverage could be added here, to help setup coverage directories and run tests with correct arguments.